ui/raidboss/data/05-shb/trial/diamond_weapon-ex.txt

```diff
@@ -0,0 +1,116 @@
+0 "Start" sync /Engage!/ window 1,1
+21.3 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/ window 25,10
+
+# Adamant Purge is a mini-phase that will be either stack, spread, or "nothing"
+# 5FA2 = Spread (Homing Laser, 5FA0)
+# 5F9B = "nothing" (Diamond Rain, 5FA7)
+# 5FA5 = Stack (Diamond Flash, 5FA1)
+# This mechanic repeats in phase 3 with additional bits and an extra tracking If circle aoe
+# @TODO: Someone better at timelines than me, sync these to mini-phases instead
+36.5 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/ window 20,5
+46.7 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+49.9 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+67.1 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+83.4 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/
+93.6 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+96.8 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+104.8 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/
+120.0 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/
+130.2 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+133.7 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+151.4 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+
+# Phase transition
+167.7 "Code Chi-Xi-Stigma" sync /:The Diamond Weapon:5FAD:/
+183.3 "Auri Cyclone" sync /:The Diamond Weapon:5FB9:/
+184.3 "Auri Cyclone" sync /:The Diamond Weapon:5FD1:/
+188.0 "Diamond Shot" sync /:Articulated Bit:5FC0:/
+
+### Phase 2
+210.0 "Outrage" sync /:The Diamond Weapon:5FBC:/
+218.2 "Outrage" sync /:The Diamond Weapon:5FBC:/
+229.6 "Auri Arts" sync /:The Diamond Weapon:5FAF:/
+236.4 "Auri Arts" sync /:The Diamond Weapon:5FB2:/
+250.4 "Outrage" sync /:The Diamond Weapon:5FBC:/
+257.6 "Auri Doomstead" sync /:The Diamond Weapon:5FBD:/
+271.8 "Auri Arts" sync /:The Diamond Weapon:5FB5:/
+272.8 "Auri Arts" sync /:The Diamond Weapon:5FCF:/
+277.0 "Vertical Cleave" sync /:The Diamond Weapon:5FB7:/
+278.7 "Vertical Cleave" sync /:The Diamond Weapon:5FD0:/
+291.1 "Outrage" sync /:The Diamond Weapon:5FBC:/
+301.3 "Articulated Bits" sync /:The Diamond Weapon:5FC1:/
+312.4 "Aetherial Bullet" sync /:Articulated Bit:5FAB:/
+312.9 "Auri Arts" sync /:The Diamond Weapon:5FAF:/
+319.7 "Auri Arts" sync /:The Diamond Weapon:5FB2:/
+320.4 "Aetherial Bullet" sync /:Articulated Bit:5FAB:/
+333.7 "Outrage" sync /:The Diamond Weapon:5FBC:/
+340.9 "Auri Doomstead" sync /:The Diamond Weapon:5FBD:/
+353.1 "Articulated Bits" sync /:The Diamond Weapon:5FC1:/
+360.7 "Aetherial Bullet" sync /:Articulated Bit:5FAB:/
+362.2 "Auri Arts" sync /:The Diamond Weapon:5FB5:/
+364.0 "Auri Arts" sync /:The Diamond Weapon:5FCF:/
+368.2 "Vertical Cleave" sync /:The Diamond Weapon:5FB7:/
+369.9 "Vertical Cleave" sync /:The Diamond Weapon:5FD0:/
+370.7 "Aetherial Bullet" sync /:Articulated Bit:5FAB:/
+382.4 "Outrage" sync /:The Diamond Weapon:5FBC:/
+
+### Phase 3
+422.7 "Flood Ray 1" sync /:The Diamond Weapon:5FC7:/
```

Comment:
It looks like you're skipping the tank lasers here and elsewhere.  Including every hit of them is maybe too noisy, but maybe there could be one line with a duration where they happen?  That would probably help tanks plan their mitigation around them better when looking at the timeline.

Reply:
The tank lasers are autos, and I've seen some inconsistencies in timing elsewhere in the fight, that's why I excluded them. Inconsistencies might just be jitter due to my east-coast ping though.

Reply:
No worries.  I think it's something that can be added with a TODO.  Even if it's not precise and there aren't syncs, it'd be good to add those lines in eventually.